app/src/main/java/org/oppia/app/onboarding/OnboadingSlideViewModel.kt

```diff
@@ -22,25 +25,29 @@ class OnboardingSlideViewModel(val context: Context, viewPagerSlide: ViewPagerSl
   private fun initializingSlide(viewPagerSlide: ViewPagerSlide) {
     when (viewPagerSlide) {
       ViewPagerSlide.SLIDE_0 -> {
-        slideImage.set(R.drawable.ic_onboarding_0)
+        if(orientation == Configuration.ORIENTATION_LANDSCAPE){slideImage.set(R.drawable.ic_landscape_onboarding_2)}
+        else if(orientation == Configuration.ORIENTATION_PORTRAIT){slideImage.set(R.drawable.ic_onboarding_0)}
```

Comment:
Write this if else code in this format:
```
       if (orientation == Configuration.ORIENTATION_LANDSCAPE) {
          slideImage.set(R.drawable.ic_landscape_onboarding_2)
        } else if (orientation == Configuration.ORIENTATION_PORTRAIT) {
          slideImage.set(R.drawable.ic_onboarding_0)
        }
```

Reply:
@Sarthak2601 Once you have solved this, please reply to this comment. In this case you can simply write `Done`, ditto for other comments too.

Reply:
Okay 👍
Done :-)